CHANGES.rst

```diff
@@ -1,3 +1,589 @@
+  file/directory exists, as long as the package and module do. [#11006]
+
+- Deprecated ``astropy.utils.OrderedDescriptor`` and
+  ``astropy.utils.OrderedDescriptorContainer``, as new features in Python 3
+  make their use less compelling. [#11094, #11099]
+
+- ``astropy.utils.masked`` provides a new ``Masked`` class/factory that can be
+  used to represent masked ``ndarray`` and all its subclasses, including
+  ``Quantity`` and its subclasses.  These classes can be used inside
+  coordinates, but the mask is not yet exposed.  Generally, the interface should
+  be considered experimental. [#11127, #11792]
+
+- Add new ``utils.parsing`` module to with helper wrappers around ``ply``. [#11227]
+
+- Change the Time and IERS leap second handling so that the leap second table is
+  updated only when a Time transform involving UTC is performed. Previously this
+  update check was done the first time a ``Time`` object was created, which in
+  practice occured when importing common astropy subpackages like
+  ``astropy.coordinates``. Now you can prevent querying internet resources (for
+  instance on a cluster) by setting ``iers.conf.auto_download = False``. This can
+  be done after importing astrpoy but prior to performing any ``Time`` scale
+  transformations related to UTC. [#11638]
+
+astropy.wcs
+^^^^^^^^^^^
+
+- Add IVOA UCD mappings for some FITS WCS keywords commonly used in solar physics. [#10965]
+
+- Add ``STOKES`` FITS WCS keyword to the IVOA UCD mapping. [#11236]
+
+- Updated bundled version of WCSLIB to version 7.6. See
+  https://www.atnf.csiro.au/people/mcalabre/WCS/CHANGES for a list of
+  included changes. [#11549]
+
+API Changes
+-----------
+
+astropy.coordinates
+^^^^^^^^^^^^^^^^^^^
+
+- For input to representations, subclasses of the class required for a
+  given attribute will now be allowed in. [#11113]
+
+- Except for ``UnitSphericalRepresentation``, shortcuts in representations now
+  allow for attached differentials. [#11467]
+
+- Allow coordinate name strings as input to ``SkyCoord.is_transformable_to`` [#11552]
+
+astropy.cosmology
+^^^^^^^^^^^^^^^^^
+
+- Change ``z_at_value`` to use ``scipy.optimize.minimize_scalar`` with default
+  method ``Brent`` (other options ``Bounded`` and ``Golden``) and accept 
+  ``bracket`` option to set initial search region. [#11080]
+
+- Clarified definition of inputs to ``angular_diameter_distance_z1z2``.
+  The function now emits ``AstropyUserWarning`` when ``z2`` is less than ``z1``. [#11197]
+
+- Split cosmology realizations from core classes, moving the former to new file
+  ``realizations``. [#11345]
+
+- Since cosmologies are immutable, the initialization signature and values can
+  be stored, greatly simplifying cloning logic and extending it to user-defined
+  cosmology classes that do not have attributes with the same name as each
+  initialization argument.
+  Also, the class abstraction is moved from ``FLRW`` to the base, ``Cosmology``. [#11515]
```

Comment:
break it into two entries rather than use "Also" in the same entry

Reply:
After thinking about how to do that I realized the "also" is really an implementation detail anyway.